webpack.config.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path');
 
 module.exports = {
+  mode: 'production',
```

Comment:
The webpack config is just for development so I believe `mode: 'production'` Unless there is a reason you added it?

Reply:
There is no special reason, just got an error or warning because the `mode` option was not set, so added it with the default value for `mode` (https://webpack.js.org/concepts/mode/#usage).

Changed it to `development` in https://github.com/orangemug/mapbox-gl-to-blob/pull/3/commits/19c777c717989a6d9649932916b1836d8e5f9dfb.